### DISCUSSION TITLE
Wrong languages in repository

### DISCUSSION THREAD

**Discussion by erem1t0** [github-linguist/linguist#7154] [Classification] [answered]
Hello! I have a problem: github says that my repository is 70% HTML(i've never seen this problem before), but i think there're no more than few html fiels. This is important for me, because i want to getting job and my resume is C++ dev. But if the employer sees, that my repo is mostly HTML, it will be bad.
There's screenshots (on the 1st you can see the statistic, on the 2nd - page when i pressed on the "html" )
![image](https://github.com/user-attachments/assets/7e01d893-1a07-4b66-bbd3-8cd84e119ea9)
![image](https://github.com/user-attachments/assets/82ef6573-5f86-4e77-98d9-5b5f38fa1a9e)
Hope you will help me, thank you!

**Comment by lildude** [comment, marked as answer]
You are seeing this because your repo has two massive HTML files:

```
HTML:
University/5 sem/numerical methods/lab1/eps_n.html
University/5 sem/numerical methods/lab1/lg_eps_n.html
```

```shell
➜ ls -lh MyGit/**/*.html
Permissions Size Date Modified Name
.rw-r--r--@ 4.6M  9 Dec 10:48  'MyGit/University/5 sem/numerical methods/lab1'/eps_n.html
.rw-r--r--@ 4.6M  9 Dec 10:48  'MyGit/University/5 sem/numerical methods/lab1'/lg_eps_n.html
➜ 
```

Linguist assesses usage by bytes of code and these two files alone account for nearly 70% of all code in your repo.

You can either remove these files or use an [override](https://github.com/github-linguist/linguist/blob/main/docs/overrides.md) to tell Linguist to ignore them.
